Makefile

```diff
@@ -1,15 +1,22 @@
 SHELL := /bin/bash
 DEST := result
 PORT := 5000
+ifeq ($(LINKS), SERVE)
+BASEURL := http://localhost:$(PORT)/
+endif
 VERSION_LINKS := 3.4 3.3 3.2 3.1 3.0 2.5 2.4
```

Comment:
We should probably extend the procedure "how to add a new Foreman version" to ensure this list is also extended with 3.5.